docs/onboarding/04-noisy-neighbor-multi-tenancy.md

```diff
@@ -0,0 +1,36 @@
+There are 4 discrete steps or considerations when initially building a multi-tenanted solution. 
+1. **Onboarding**. For this step, we consider how tenants get introduced into your environment. How will they sign up? How will whatever resources they need be instantiated?
+1. **Authentication**. From the moment a user enters our environment, they should be associated with an identity that follows every action they take. An example of this would be a SAS token which is carried along every step of the architectural flow, so that entities can authenticate into whatever microservices we are using.  
+1. **Tenant Isolation**. Related, but not equivalent to authentication, this step essentially asks the question: “how do we ensure tenants cannot reach across boundaries?” This can be done in two ways. The first is through siloed infrastructure, wherein resources are divided into physical partitions and each tenant gets its own environment to work in. A silo model eliminates the noisy neighbor problem, but it is more difficult to scale and typically much more expensive. A more modern multi-tenancy pattern uses the pool model, wherein resources and microservices are shared across tenants but, via runtime policies and role-based access control, you ensure tenants cannot reach across boundaries. 
+1. **Application**. This is the central part of your architecture. Ideally, by this point you will not need to think about multi-tenancy much. If you have designed your architecture to support tenant isolation and authorization/authentication, from the developer perspective it’s just a matter of figuring out how many instances of your application and other resources you need to provision.
+
+## Key Points
+- The 'Noisy Neighbor' problem refers to when resources are shared, and one user takes up more of the resources than allocated. As a result, one or more other users are impacted. 
+- When designing a multi-tenant architecture, the silo model can: lead to over-provisioned resources, make it difficult to scale, and spike cost. However, it has the advantage of physical isolation, which can increase tenant confidence that their data is safe as well as eliminated the noisy neighbor problem. 
+- The pool model allows for cost-efficient elastic scaling, and by implementing logical separations of resources rather than physical, you can then still enforce RBAC as needed. However, it has the drawback of being more complex to design, as those logical separations require a careful and accurate approach, and you also have to find workarounds for the classic 'Noisy Neighbor' problem.
+
+## Learning Exercises
+
+### Grant a user access to resources using the Portal or PowerShell
+RBAC allows you to grant users access to specifically scoped resources, from subscriptions to a particular container. [Follow this tutorial to get familiarized with RBAC](https://docs.microsoft.com/en-us/azure/role-based-access-control/tutorial-role-assignments-user-powershell).
```

Comment:
Is there a way we can root this in ADT? For example, should we have them grant a reader role to a user and prove that user cannot modify models or twins?